lesson9/index.js

```diff
@@ -0,0 +1,40 @@
+const ul = document.getElementById('js-lists');
+
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+async function promise() {
```

Comment:
promise as function name  is not good. because JavaScript has API of Promise already. Reader misidentifies. so Think of another name.

Reply:
I understand.
I named `loadingPromise`.
commit: ce38c46